cmd/gmailctl/cmd/init_cmd.go

```diff
@@ -44,6 +52,8 @@ func init() {
 	// Flags and configuration settings
 	initCmd.Flags().BoolVar(&initReset, "reset", false, "Reset the configuration.")
 	initCmd.Flags().BoolVar(&initRefreshExpired, "refresh-expired", false, "Refresh auth token if expired.")
+	initCmd.Flags().BoolVar(&initResetLib, "reset-lib", false, "Reset the library file.")
```

Comment:
I would call this flag `update-lib`. It's true that we are resetting it to the default, but the default may have changed since the time the configuration was initialized. It would be less confusing.